variables-workers.tf

```diff
@@ -230,3 +230,21 @@ variable "fss_max_fs_stat_files" {
   description = "Maximum tfiles, ffiles, and afiles values reported by NFS FSSTAT."
   type        = number
 }
+
+variable "platform_config" {
```

Comment:
Please don't use object as variable type in root.

Reply:
As discussed, the object type isn't what it used to be with `optional` field support now - this provides better type specification for fields within, without needing to require them all as before.